docs/BUILD

```diff
@@ -9,20 +9,24 @@ exports_files(["protodoc_manifest.yaml"])
 
 envoy_package()
 
-# TODO(phlax): fix failing/excluded configs
-# the following config only fails on windows:
-#       dns-cache-circuit-breaker: "Error: unable to read file: /etc/ssl/certs/ca-certificates.crt"
-
 filegroup(
     name = "configs",
     srcs = glob(
         ["root/**/*.yaml"],
         exclude = [
+            # `some.customer.filter`
             "root/intro/_include/life-of-a-request.yaml",
```

Comment:
i guess its not trivial to add `some.customer.filter` - wondering how we can keep this config valid

Reply:
any suggestions on which filter to use would be great

i think we are going to need to look at the docs related to these configs to see what effect these changes have

Reply:
i have commented out the `some.customer.filter` example from the filter chain - as i think it still serves its purpose being present but commented out - open to suggestions

Reply:
I would prefer it to be included, since it's designed to capture what happens when you add a customer filter and is referenced extensively in the text (as "CustomerFilter"). I wonder if we can somehow augment `envoy/tools/type_whisperer/all_protos_with_ext_pb_text.pb_text` to reflect some dummy config.